py4DSTEM/process/utils/ellipticalCoords.py

```diff
@@ -407,15 +264,19 @@ def radial_integral(ar, x0, y0, dr):
 
 def radial_elliptical_integral(ar, dr, ellipse_params):
     """
-    Computes the radial integral of array ar from center (x0,y0) with a step size in r of dr.
+    Computes the radial integral of array ar from center (x0,y0) with a step size in r of
```

Comment:
Should both this function and radial_integral exist? Why not have radial_elliptical_integral have default values such that it is just the circular radial integral? Would prevent having to update the same code twice.

I suggest it looking like radial_elliptical_integral(ar, dr, x0, y0, ellipse_params=(1,1,0))

Reply:
I think having both fns is useful from a user-facing perspective - `radial_integral` is conceptually simpler than `radial_elliptical_integral`, and having the option of just calling a regular old radial integral is nice.  That said, I 100% agree that the duplicated code is not what we want.  So I left the `radial_integral` fn call signature and docstring, but then changed its code to one line that just calls `radial_elliptical_integral` with circular ellipse params.  Lmk if you have any concerns with this, I think/hope it covers our bases